pom.xml

```diff
@@ -84,7 +84,7 @@
     </dependency>
     <dependency>
       <groupId>io.cdap.cdap</groupId>
-      <artifactId>cdap-data-pipeline</artifactId>
+      <artifactId>cdap-data-pipeline2_2.11</artifactId>
```

Comment:
why we added this?

Reply:
Bcoz it was not supporting the cdap version 6.6.0.

Reply:
adding 2_2.11 doesn't make much sense, can you test without this.